src/main.cc

```diff
@@ -92,7 +101,6 @@ static std::unique_ptr<Uptane::Target> find_target(LiteClient& client,
 }
 
 static data::ResultCode::Numeric do_update(LiteClient& client, Uptane::Target target) {
-  target.InsertEcu({client.primary_ecu.first, client.primary_ecu.second});
```

Comment:
Have you tested that update events will still get reported to the server with this removed? Now that we have our own "TufClient" its probably fine, but in the old SotuptaneClient approach, this was required.

Reply:
I'll double check